tests/tests/lightspeed-install.cy.ts

```diff
@@ -26,11 +26,30 @@ describe('Lightspeed related features', () => {
     cy.adminCLI(
       `oc adm policy add-cluster-role-to-user cluster-admin ${Cypress.env('LOGIN_USERNAME')}`,
     );
-    cy.login(
-      Cypress.env('LOGIN_IDP'),
-      Cypress.env('LOGIN_USERNAME'),
-      Cypress.env('LOGIN_PASSWORD'),
-    );
+    // Getting the oauth url for hypershift cluster login
+    cy.exec(
+      `oc get oauthclient openshift-browser-client -o go-template --template="{{index .redirectURIs 0}}" --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
+    ).then((result) => {
+      if (expect(result.stderr).to.be.empty) {
```

Comment:
Probably don't need the `if` since `expect` is like an assertion that will fail the test if stderr is not empty?

Reply:
I got it into an if because linting was complaining of a loose expect. So i included it in the if to get rid of the error